src/jats/importer/jats-transformations.ts

```diff
@@ -477,6 +477,13 @@ export const moveReferencesToBackmatter = (
   backmatter.appendChild(section)
 }
 
+export const moveHistoryToBody = (document: Document, body: Element) => {
```

Comment:
Do we need to move the dates? can we read them directly from their original location?

Reply:
Prior to the changes in LEAN-3911 it was necessary, but it is no longer needed, so i updated the PR